frontend/src/components/mine/Tailings/MineTailingsFilePicker.js

```diff
@@ -27,15 +27,20 @@ class MineTailingsFilePicker extends Component {
     this.props.fetchMineDocuments(this.props.selectedDocument.mine_guid);
   }
 
-  handleFileSelect = (mineDocumentGuid) => {
-    const data = { mine_document_guid: mineDocumentGuid };
-    this.props
-      .addMineDocumentToExpectedDocument(this.props.selectedDocument.exp_document_guid, data)
-      .then(() => this.refreshUploadedFiles());
+  handleFileSelect = (mine_document_guid) => {
+    const data = { mine_document_guid };
+    this.pushDocument(data);
+  };
```

Comment:
Personal preference... I'd remove these `data` variables since they're not necessary

Reply:
Yeah these were kinda leftover from a small refactor, I'll update this.